GRC: Generates bare `except:` code, which our own coding standards frown upon

### What happened?

When I run the same command we use in the CI for upstream code contributions on GRC-generated python programs I get errors about E722 – bare except.

We should probably generate code that passes by our quality standards!

### System Information

source built of current main

### GNU Radio Version

4.0-git (dev-4.0)

### Specific Version

0.0.0.0

### Steps to Reproduce the Problem

`pycodestyle --max-line-length 120 generated_fg.py`


### Relevant log output

```shell
./generated_fg.py:45:9: E722 do not use bare 'except'

(amongst other warnings)
```

--- a/grc/core/generator/flow_graph.py.mako
+++ b/grc/core/generator/flow_graph.py.mako
@@ -82,8 +82,8 @@ class ${class_name}(gr.top_block, Qt.QWidget):
         qtgui.util.check_set_qss()
         try:
             self.setWindowIcon(Qt.QIcon.fromTheme('gnuradio-grc'))
-        except:
-            pass
+        except BaseException as exc:
+            print(f"Qt GUI: Could not set Icon: {str(exc)}", file=sys.stderr)
         self.top_scroll_layout = Qt.QVBoxLayout()
         self.setLayout(self.top_scroll_layout)
         self.top_scroll = Qt.QScrollArea()
@@ -103,8 +103,8 @@ class ${class_name}(gr.top_block, Qt.QWidget):
                 self.restoreGeometry(self.settings.value("geometry").toByteArray())
             else:
                 self.restoreGeometry(self.settings.value("geometry"))
-        except:
-            pass
+        except BaseException as exc:
+            print(f"Qt GUI: Could not restore geometry: {str(exc)}", file=sys.stderr)
 % elif generate_options == 'bokeh_gui':
 
 class ${class_name}(gr.top_block):

